webextensions/javascript-apis.json

```diff
@@ -5530,7 +5530,10 @@
                     "version_added": "26"
                   },
                   "edge": {
-                    "version_added": true
+                    "version_added": true,
+                    "notes": [
+                      "The `tab` object is incomplete."
```

Comment:
I think @scottlow is probably also a good candidate for this question. ;)

And while on it, what is the following supposed to mean?

> `runtime.sendmessage` is not supported from site pages

I didn't even bother to document it because I have literally no idea what it means. :D